src/mount/mount.ceph.c

```diff
@@ -912,7 +912,8 @@ static int do_mount(const char *dev, const char *node,
 			fprintf(stderr, "mount error: ceph filesystem not supported by the system\n");
 			break;
 		case EHOSTUNREACH:
-			fprintf(stderr, "mount error: no mds server is up or the cluster is laggy\n");
+			fprintf(stderr, "mount error: no mds (Metadata Server) is up. "
+			"The cluster might be laggy, or you may lack authorization\n");
```

Comment:
nit: "... or you may not be authorized."

Reply:
oh, the pr isn't updated with the change.

Reply:
Actually, earlier it was `...or you may not be authorized` but after dhairya's suggestion  changed to `..may lack authorization`. I am on the fence here. If you really want, I will update:)

Reply:
@vshankar updated with your suggestion.